tests/pytests/test_juno_drivers.py

```diff
@@ -121,15 +21,14 @@ def test_kernels():
     for kern in binary_kernels:
         os.remove(kern)
 
-@pytest.mark.xfail
+# @pytest.mark.xfail
 @pytest.mark.parametrize("label_type", ['isis3'])
-@pytest.mark.parametrize("formatter", ['isis'])
-def test_mro_load(test_kernels, label_type, formatter, isis_compare_dict):
+def test_mro_load(test_kernels, label_type):
     label_file = get_image_label('JNCR_2016240_01M06152_V01', label_type)
-
-    usgscsm_isd_str = ale.loads(label_file, props={'kernels': test_kernels}, formatter=formatter)
+    usgscsm_isd_str = ale.loads(label_file, props={'kernels': test_kernels})
     usgscsm_isd_obj = json.loads(usgscsm_isd_str)
-    print(json.dumps(usgscsm_isd_obj, indent=4))
+    isis_compare_dict = get_isd('juno')
+    print(json.dumps(usgscsm_isd_obj, indent=2))
```

Comment:
print statement here -- deliberately left in?

Reply:
Yeah we leave these in because they help when diagnosing failures. If the test passes they are just swallowed.